docs/content/entity-templates.md

```diff
@@ -13,13 +13,13 @@ For information on how to create SpatialOS entities once you have set up entity
 
 You have to create an `EntityTemplate` to specify which components a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity) has and the initial values of those [components]({{urlRoot}}/content/glossary#spatialos-component). You have to also specify which workers have write access (also known as ”[authority]({{urlRoot}}/content/glossary#authority)” on a per-component basis.
 
-You use the [`EntityBuilder` class]({{urlRoot}}/content/api-entity-builder) to do this.
+You use the `EntityTemplate` class.
 
 There are examples of how to use this class below.
 
 
-### Create `ComponentData`
-For each [schema component]({{urlRoot}}/content/glossary#schema) you define, the class created by the [code generator]({{urlRoot}}/content/code-generator) has a method to create a `ComponentData` object for that component. For example, for the following schema:
+### Create component snapshots
+For each [schema component]({{urlRoot}}/content/glossary#schema) you define, the [code generator]({{urlRoot}}/content/code-generator) generates a struct which inherits from `ISpatialComponentSnapshot`. For example, for the following schema:
```

Comment:
```suggestion
For each [schema component]({{urlRoot}}/content/glossary#schema) you define, the [code generator]({{urlRoot}}/content/code-generator) creates a struct which inherits from `ISpatialComponentSnapshot`, the generated [snapshot struct]({{urlRoot}}/content/ecs/component-generation#overview). For example, for the following schema:
```
Added a link to where you can find info on `ISpatialComponentSnapshot`. As it's not entirely clear.